### DISCUSSION TITLE
How does this Compare to Storm?

### DISCUSSION THREAD

**Discussion by GoudaCouda** [assafelovic/gpt-researcher#999] [General]
I seen that this and storm(Similar to this from Stanford) are sort of the same idea but im assuming there are big technical difference. Is there anything that either are better at or any ideas we could use from Storm for this code?

**Comment by ElishaKay** [comment]
You can [join the discord](https://discord.gg/QgZXvJAccX) to [see Assaf's answer here](https://discord.com/channels/1127851779011391548/1127851779573420053/1310848409510613032):

`
GPT Researcher is aimed at a much broader and in depth research process, also focusing on UX and customization for developers. STORM takes an opinionated approach at generating research with a wiki like output. We've actually took some inspiration from them and have implemented some of their strategies at GPTR. I like to see GPTR as the best of both worlds 😉
`
